pkg/identity/numericidentity.go

```diff
@@ -365,6 +369,15 @@ func InitWellKnownIdentities(c Configuration) int {
 	return len(WellKnown)
 }
 
+// InitClusterIDShift sets variables that control the bit allocation of cluster
+// ID in a numeric identity.
+func InitClusterIDShift() {
+	// ClusterIDLen is the number of bits that represent a cluster ID in a numeric identity
+	ClusterIDLen = int(math.Log2(float64(cmtypes.ClusterIDMax + 1)))
+	// ClusterIDShift is the number of bits to shift a cluster ID in a numeric identity
+	ClusterIDShift = NumericIdentityBitlength - ClusterIDLen
```

Comment:
rather than a panic, i added Getters that initialize the variables the 1st time one of them is used. what do you think about this approach? (just realized i need to update the commit msg)